uswds/uswds-site
# USWDS-Site - Utility settings: Update maps instructions under state and responsive variants

# Summary

**Update sass map instructions under utility state and responsive variant sections.** Instructions match pattern introduced in https://github.com/uswds/uswds/pull/5216

## Related issue

Closes #2076

## Preview link

[Configure settings maps | Settings page →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-update-sass-map-instructions/documentation/settings/#configuring-settings-maps-2)

[Utilities settings →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-update-sass-map-instructions/utilities/#settings-2)

[Responsive + State variants →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-update-sass-map-instructions/utilities/color/#responsive-variants-2)

## Problem statement

1. The updated sass map pattern allows users to only update settings they _want to change_. All others maintain default values set by USWDS.
2. The existing instructions told users they had to define _all_ map elements and _not_ just the elements they wish to change.

## Solution

Update documentation to match the new pattern. The new instructions read:

> When modifying any settings stored as a map, undefined elements of the map will take its default value.


## Testing and review

1. Visit any utility page with responsive & state variants, such as color
2. Read instructions under responsive & state variants.
3. Confirm they have been updated with the new pattern instruction.
4. Confirm instructions are clear & concise. 

### Further consideration

This update only updates existing map instructions. I think it might be beneficial to users to add additional sass map instructions on the [settings](https://designsystem.digital.gov/documentation/settings/) and [utilities](https://designsystem.digital.gov/utilities/) pages.


Before opening this PR, make sure you’ve done whichever of these applies to you:
- [ ] Run `npm test` and confirm that all tests pass. _// Waiting for snyk update_
